ibis/tests/expr/test_value_exprs.py

```diff
@@ -1031,53 +1026,50 @@ def test_scalar_parameter_invalid_compare(left, right):
 )
 def test_between_time_failure_time(case, creator, left, right):
     value = creator(case)
-    with pytest.raises(TypeError):
+    with pytest.raises(ValidationError):
         value.between(left, right)
 
 
-def test_custom_type_binary_operations():
-    class Foo(ir.Expr):
-        def __add__(self, other):
-            op = self.op()
-            return type(op)(op.value + other).to_expr()
+# def test_custom_type_binary_operations():
```

Comment:
Does this need to be uncommented or removed?

Reply:
I'd rather remove it.

Reply:
Removed it.